src/libraries/ErrorsLib.sol

```diff
@@ -24,6 +24,7 @@ library ErrorsLib {
     /// @notice Thrown when the proof is invalid or expired when claiming rewards.
     string internal constant INVALID_PROOF_OR_EXPIRED = "invalid proof or expired";
 
-    /// @notice Thrown when rewards have already been claimed.
-    string internal constant ALREADY_CLAIMED = "already claimed";
+    /// @notice Thrown when the Merkle tree contains a node with fewer rewards than what it has previously claimed. Also
+    /// thrown if user claims twice with the same root.
+    string internal constant CLAIMABLE_TOO_LOW = "claimable too low";
```

Comment:
[optional] I would switch the 2 sentences because claiming twice is a user error, so more likely to be relevant